gem/lib/pupperware/spec_helper.rb

```diff
@@ -136,6 +150,14 @@ def teardown_cluster
   # Docker Helpers
   ######################################################################
 
+  def pull_images(ignore_service)
```

Comment:
Since this relies on `docker_compose` it should probably be in the `Docker Compose Helpers` section, I think.

Not a big deal though.

I imagine that we'll eventually move each section into its own .rb file and then just import them all in this file.

But that's easy to reorganize stuff if/when we get there.